tests/functional/aws-node-sdk/test/legacy/testBucketVersioning.js

```diff
@@ -0,0 +1,131 @@
+import assert from 'assert';
```

Comment:
These aren't legacy tests, right?

Also, tests are not self-contained. (They should be able to run with an `.only()`)